packages/dapp/src/providers/MenuProvider.tsx

```diff
@@ -95,6 +87,17 @@ const MenuProvider = ({ children }: MenuProviderProps) => {
     }));
   };
 
+  // Toggle Navbar Chains Menu
+  const onOpenNavbarChainsMenu = (open: boolean) => {
+    setMenu((oldSettings) => ({
+      ...oldSettings,
+      openNavbarSubMenu: open
+        ? SubMenuKeys.chains
+        : (SubMenuKeys.none as string),
+      openNavbarChainsMenu: open as boolean,
```

Comment:
`open` already is being expected as boolean, this assertion isn't required :)

Reply:
Good point! Had this all over the place, now removed